src/drivers/github.go

```diff
@@ -24,3 +32,65 @@ func (driver GithubCodeHostingDriver) GetNewPullRequestURL(repository string, br
 func (driver GithubCodeHostingDriver) GetRepositoryURL(repository string) string {
 	return "https://github.com/" + repository
 }
+
+// MergePullRequest merges the pull request through the Github API
+func (driver GithubCodeHostingDriver) MergePullRequest(options MergePullRequestOptions) error {
+	ts := oauth2.StaticTokenSource(
+		&oauth2.Token{AccessToken: os.Getenv("GITHUB_TOKEN")},
+	)
+	tc := oauth2.NewClient(context.Background(), ts)
```

Comment:
Or even better a Connect method that stores the connection as state of the object